.travis.yml

```diff
@@ -10,18 +10,21 @@ rvm:
 - &oldest_ruby 2.3.8
 - jruby-9.2.7.0
 - jruby-9.1.17.0
-# the test suite currently crashes on truffleruby
-#- truffleruby-rc13
 matrix:
   include:
   - rvm: *oldest_ruby
     env: ROUGE_VERSION='~> 2.0.0'
+  - name: "TruffleRuby"
+    rvm: truffleruby-19.1.0
+    if: env(TRAVIS_EVENT_TYPE) = cron
 env:
   global:
   # use system libraries to speed up installation of nokogiri
   - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   - PYGMENTS_VERSION='~> 1.2.0'
   - SOURCE_DATE_EPOCH=1521504000
+  # use 3g to avoid OutOfMemoryError when running the tests suite against TruffleRuby
+  - TRUFFLERUBYOPT="--vm.Xmx3g"
```

Comment:
Perhaps just define this environment variable in the matrix entry?

Reply:
I didn't know it was possible, fixing...